frontend/pages/SoftwarePage/SoftwareVulnerabilities/SoftwareVulnerabilitiesTable/VulnerabilitiesTableConfig.tsx

```diff
@@ -0,0 +1,276 @@
+      Header: (headerProps: IHeaderProps): JSX.Element => {
+        const titleWithToolTip = (
+          <TooltipWrapper
+            tipContent={
+              <>
+                The date this vulnerability was published in the National
+                Vulnerability Database (NVD).
+              </>
+            }
+          >
+            Published
+          </TooltipWrapper>
+        );
+        return (
+          <>
+            <HeaderCell
+              value={titleWithToolTip}
+              isSortedDesc={headerProps.column.isSortedDesc}
+            />
+            {isSandboxMode && <PremiumFeatureIconWithTooltip />}
+          </>
+        );
+      },
+      Cell: ({ cell: { value } }: ITextCellProps): JSX.Element => {
+        const valString = typeof value === "number" ? value.toString() : value;
+        return (
+          <TextCell
+            value={valString ? { timeString: valString } : undefined}
+            formatter={valString ? HumanTimeDiffWithDateTip : undefined}
+          />
+        );
+      },
+    },
+    {
+      title: "Detected",
+      accessor: "created_at",
+      disableSortBy: false,
+      Header: (headerProps: IHeaderProps): JSX.Element => {
+        return (
+          <>
+            <HeaderCell
+              value="Detected"
+              isSortedDesc={headerProps.column.isSortedDesc}
+            />
+            {isSandboxMode && <PremiumFeatureIconWithTooltip />}
+          </>
+        );
+      },
+      Cell: (cellProps: ICellProps): JSX.Element => {
+        const createdAt = cellProps.row.original.created_at || "";
+
+        return (
+          <TextCell
+            value={{ timeString: createdAt }}
+            formatter={HumanTimeDiffWithDateTip}
+          />
+        );
+      },
+    },
+    {
+      title: "Hosts",
+      Header: (cellProps: IHeaderProps): JSX.Element => (
+        <HeaderCell
+          value="Hosts"
+          disableSortBy={false}
+          isSortedDesc={cellProps.column.isSortedDesc}
+        />
+      ),
+      disableSortBy: false,
+      accessor: "hosts_count",
+      Cell: (cellProps: ITextCellProps): JSX.Element => {
+        const { hosts_count } = cellProps.row.original;
+        return <TextCell value={hosts_count} />;
+      },
+    },
+    {
+      title: "",
+      Header: "",
+      accessor: "linkToFilteredHosts",
+      disableSortBy: true,
+      Cell: (cellProps: ICellProps) => {
+        return (
+          <>
+            {cellProps.row.original && (
+              <ViewAllHostsLink
+                queryParams={{
+                  vulnerability: cellProps.row.original.cve,
+                }}
+                className="vulnerabilities-link"
+                rowHover
+              />
+            )}
+          </>
+        );
+      },
+    },
+  ];
+
+  if (!isPremiumTier) {
+    return tableHeaders.filter(
+      (header) =>
+        header.accessor !== "epss_probability" &&
+        header.accessor !== "cve_published"
```

Comment:
nit, I find something like `!["s1", "s2"].includes(header.accessor)` to be a little cleaner for this kind of thing